torch/csrc/jit/backends/nnapi/nnapi_backend_preprocess.cpp

```diff
@@ -0,0 +1,48 @@
+#include <pybind11/pybind11.h>
+#include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/backends/backend_preprocess.h>
+#include <torch/csrc/jit/python/pybind_utils.h>
+
+namespace py = pybind11;
+
+// Converts model to nnapi and serializes it for mobile
+// Returns a dictionary string with one entry:
+// Key: "NnapiModule"
+// Value: a string of the nnapi module, saved for mobile
+//
+// method_compile_spec should contain an input Tensor with the following format:
```

Comment:
A few questions:
* Does the delegate will only work for "forward" and no other methods? Is there a reason why?
* The value is itself another dictionary, which must have an entry for "inputs" with a value of Tensor type (and shape? and what else?) that matches that of the inputs in the method.

Reply:
* I remember @dreiss mentioned that nnapi itself has "forward" as the only entry point. 
* @Amyh11325 is working on a sanity check of the compilation spec. Currently she's trying  to support both a tensor and a list of tensors.

Reply:
Sounds good. Just checking, and also worth documenting :-)